### ISSUE TITLE
Adding semi real-time cababilities

### ISSUE THREAD

**Issue by ldcouto** [into-cps/intocps-ui#91]
<a href="https://github.com/mpcagro"><img src="https://avatars.githubusercontent.com/u/17993540?v=3" align="left" width="96" height="96" hspace="10"></img></a> **Issue by [mpcagro](https://github.com/mpcagro)**
_Monday Mar 21, 2016 at 23:21 GMT_
_Originally opened as https://github.com/into-cps/INTO-CPS_Application/issues/6_

----

I am still unsure if this should be added here or published in the forum.
Please write if this should be published on some other page.

I am missing the ability to set the co-simulation engine in semi-real mode.
I write semi since it requires the FMU are able to meet the time deadlines 
and it can be done on a non-real time operating system.
Basically the COE must follow the system clock (operating system clock), when controlling the fmu'us.

**Comment by ldcouto** [comment]
<a href="https://github.com/CThuleHansen"><img src="https://avatars.githubusercontent.com/u/3479481?v=3" align="left" width="48" height="48" hspace="10"></img></a> **Comment by [CThuleHansen](https://github.com/CThuleHansen)**
_Monday Apr 04, 2016 at 06:40 GMT_

----

This will be added in the future, but a date is not set yet.

**Comment by ldcouto** [comment]
Keeping this as an open feature request
